### PR TITLE
chore(project): add patch release checklist

### LINKED ISSUE #1520: Have a release checklist for patch releases

### PR DESCRIPTION
**Pull request by barmac**
Closes #1520

### CODE REVIEW

File: docs/.project/PATCH_RELEASE_CHECKLIST.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+```markdown
+* [ ] make sure dependencies are released (`rm -rf node_modules && npm i && npm run all` works)
+    * `bpmn-js`, `dmn-js`, `cmmn-js`, `*-properties-panel`, `*-moddle`, ...
+* [ ] close all issues which are solved by dependency updates
+* [ ] smoke test to verify all diagrams can be created
+* [ ] update CHANGELOG
+* [ ] semantic release (`npm run release`), cf. [release schema](https://github.com/bpmn-io/internal-docs/tree/master/release-schema) 
+* [ ] wait for [Travis](https://travis-ci.org/camunda/camunda-modeler) to build the release
+* [ ] execute [integration test](https://github.com/camunda/camunda-modeler/blob/master/docs/.project/INTEGRATION_TEST.md) on [released artifacts](https://github.com/camunda/camunda-modeler/releases)
```
Comment (pinussilvestrus) [inline docs/.project/PATCH_RELEASE_CHECKLIST.md]: Don't think we have to have integration testing and the qa team involved in every patch release, but let's keep this in if we want to have it in some cases.

File: docs/.project/PATCH_RELEASE_CHECKLIST.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    * [ ] Works on Linux
+    * [ ] Works on Mac
+    * [ ] Works on Windows
+* [ ] trigger QA for fuzzy testing
```
Comment (pinussilvestrus) [inline docs/.project/PATCH_RELEASE_CHECKLIST.md]: Don't think we have to have integration testing and the qa team involved in every patch release, but let's keep this in if we want to have it in some cases.

Comment (nikku) [reply docs/.project/PATCH_RELEASE_CHECKLIST.md]: We should make a short test plan (what has changed / needs to be tested) and do fuzzy testing based on it.
Thereby we should focus on the things we have fixed (and theirs surroundings).

It is a good idea to involve QA here, too, as they can find many bugs we are to blind to uncover. They need the test plan and/or fixed bugs as input. However, we should not rely on their presence for patch releases.

Comment (barmac) [reply docs/.project/PATCH_RELEASE_CHECKLIST.md]: What about if we mark it as `[optional]`?

Comment (nikku) [reply docs/.project/PATCH_RELEASE_CHECKLIST.md]: Let's drop the integration test section and mention the focussed test plan instead.

**Comment by barmac** [comment]
@nikku @pinussilvestrus ready for second round

**Comment by nikku** [comment]
Looks good to me. I've linked the list from the RELEASE_CHECKLIST, just in case.

**Pull request merged**